pkg/pbutil/output.go

```diff
@@ -64,3 +65,30 @@ func FTextPB(w io.Writer, m protoreflect.ProtoMessage) error {
 	_, err = w.Write(mt)
 	return err
 }
+
+// GeneratePBBinaryMessage2File generates binary message to the file specified by `filename`.
+func GeneratePBBinaryMessage2File(m protoreflect.ProtoMessage, filename string, fs afero.Fs) error {
```

Comment:
```suggestion
// GeneratePBBinaryMessageFile generates a binary message to the file specified by `filename`.
func GeneratePBBinaryMessageFile(m protoreflect.ProtoMessage, filename string, fs afero.Fs) error {
```
Nit: The "2" is unnecessary. If it were necessary, "To" would be preferable as well, since it's less ambiguous and jargony.